### PR TITLE
Add automated tests

### PR DESCRIPTION
**Pull request by obliviusm**
Add regression tests to compare screenshots of new and old versions

### CODE REVIEW

**Comment by obliviusm** [comment]
i need to test this on other computers
@sweatC @andriyParashchuk can you install this branch and run `yarn backstop test`?

**Comment by obliviusm** [comment]
image

**Comment by obliviusm** [comment]
@andriyParashchuk @pftg @sweatC pls review
staging http://jt-pr289.surge.sh/?height=768px

**Comment by sweatC** [comment]
@obliviusm why do you need `height` parameter?
How can I test your PR? Do I need to use `ghostinspector`?

File: _assets/javascripts/application.js [hunk 1]
```diff
@@ -194,6 +194,34 @@ $(document).ready(function () {
   document.addEventListener("scroll", lazyLoad);
   window.addEventListener("resize", lazyLoad);
   window.addEventListener("orientationchange", lazyLoad);
+
+  // fixed section height (for tests)
+  var getUrlParameter = function getUrlParameter(sParam) {
+    var sPageURL = decodeURIComponent(window.location.search.substring(1)),
```
Comment (pftg) [inline _assets/javascripts/application.js:200]: no need to use Venti style notation for vars names in the JS code. We have only one prefix `$` to indicate that this is wrapped by `jquery`.

Comment (obliviusm) [reply _assets/javascripts/application.js:200]: that's not my code. i just copied this function. i don't think we should care about re-writing it.
maybe better to move it somewhere?

**Review by pftg — APPROVED** [review]
LGTM

**Pull request merged**